component/core/src/test/java/org/exoplatform/social/core/jpa/test/AbstractCoreTest.java

```diff
@@ -108,7 +115,8 @@ protected void tearDown() throws Exception {
     deleteAllSpaces();
     deleteAllIdentitiesWithActivities();
     deleteAllActivities();
-
+    deleteAllProfileSettings();
+    deleteAllLabels();
```

Comment:
Please add this to newly introduced test classes only to not having to execute it for other test classes and thus reduce the test performances without needing it.

( Please move changes made on this abstract class test into newly introduced classes only )